platforms/hyperledger-besu/configuration/roles/create/prometheus/tasks/main.yaml

```diff
@@ -0,0 +1,29 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Fetching the prometheus URLs
+- name: Fetching the prometheus URLs
+  include_tasks: nested_loop.yaml
+  vars:
+    service: "{{ org.services.peers is defined | ternary(org.services.peers, org.services.validators) }}"
+    members: "{{service}}" 
+
+#Adding the prometheus chart repo
+- name: Add stable chart repo
+  kubernetes.core.helm_repository:
+    name: stable
+    repo_url: "https://prometheus-community.github.io/helm-charts"
+
+# Install prometheus via helmchart
+- name: Deploy latest version of Prometheus chart
```

Comment:
Can you check if a particular version of the chart can be used. Use of latest might break later